tests/pipeline_test.py

```diff
@@ -1208,3 +1208,43 @@ def process(
 def test_html_repr() -> None:
     pipeline = sl.Pipeline([make_int], params={float: 5.0})
     assert isinstance(pipeline._repr_html_(), str)
+
+
+def test_pipeline_keyword_argument_and_param(scheduler: sl.scheduler.Scheduler) -> None:
```

Comment:
Can you name this (and below) with something like `keyword_only` in the test name? I think keyword args have always been supported?